auth0_flutter/darwin/Classes/AuthAPI/AuthAPIPhoneNumberPasswordlessLoginMethod.swift

```diff
@@ -0,0 +1,46 @@
+import Auth0
+
+#if os(iOS)
+import Flutter
+#else
+import FlutterMacOS
+#endif
+
+struct AuthAPIPhoneNumberPasswordlessLoginMethod: MethodHandler {
+    enum Argument: String {
+        case phoneNumber
+        case passwordlessType
+        case connection
```

Comment:
This param here isn't really needed here as it will always be `sms`. This is something that is noted to be corrected in the next major version.